src/cockpit/channels/stream.py

```diff
@@ -84,7 +96,8 @@ def create_transport(self, loop: asyncio.AbstractEventLoop, options: Dict[str, A
 
         try:
             logger.debug('Spawning process args=%s', args)
-            return SubprocessTransport(loop, self, args, pty, window, env=env, cwd=cwd, stderr=stderr)
+            return SubprocessTransport(loop, self, args, pty, window, env=env, cwd=cwd, stderr=stderr,
+                                       preexec_fn=lambda: prctl(SET_PDEATHSIG, signal.SIGHUP))
```

Comment:
Why not `SIGKILL`?

Reply:
Good question -- hysterical raisins mostly, the C bridge [uses SIGHUP](https://github.com/cockpit-project/cockpit/blob/main/src/common/cockpitpipe.c#L1272). It's been like that forever, see commit aa407c961d98e.

SIGKILL seems a little aggressive, SIGTERM may be a good alternative.

So.. I dunno